8.0/Dockerfile

```diff
@@ -26,8 +26,20 @@ RUN docker-php-ext-configure gd \
     --with-webp
 RUN docker-php-ext-install gd
 
-RUN pecl install amqp-1.9.4
-RUN docker-php-ext-enable amqp
+# Extract PHP source
+RUN docker-php-source extract
```

Comment:
Can we run all the amp stuff in one RUN command and add a comment when to remove this code and what should be used instead then?

Reply:
this is the suggestion from @DracoBlue, but i think we should also wait for a new release amqp version. i will remove this

Reply:
@niepi please put it into one RUN command, to avoid having the entire php code in one docker build

Reply:
we probably will not release this with the master version anyway, so I remove this and wait for amqp release

Reply:
I am 👍🏻 for releasing it with the master version, it's better to test it against an unstable version and see if all your other code works well with PHP 8, instead of not be able to use this image, because of an missing extension, WDYT?